configure

```diff
@@ -1181,6 +1183,13 @@ for opt do
   # everything else has the same name in configure and meson
   --enable-* | --disable-*) meson_option_parse "$opt" "$optarg"
   ;;
+  --with-libafl-bridge=*)
```

Comment:
Why this way of linking and not like we do for usermode? In this way users must compile QEMU theirself + will be hard to expose to pylibafl

Reply:
I think @bitwave uses some forked qemu for his project, slotting in some patched full-system qemu is a use case we should support somehow

Reply:
I used the qemu repository, added @andreafioraldi 's fork as remote and used the older usermode version, where qemu is build not as a library. This way it was a drop-in replacement for qemu inside the OP-TEE project (my fuzzing target)